.gitignore

```diff
@@ -17,4 +17,4 @@ inertia.toml
 
 # Docs generation files
 _build/
-credentials/
+/credentials/
```

Comment:
wait why?

Reply:
Apparently makes it so that it only ignores the `credentials` directory in the root directory. Needed to make another one (`tests/credentials`) to test with actual `toml` files. (don't know how good this idea actually is ehe)